src/ovs/aca_ovs_l3_programmer.cpp

```diff
@@ -245,7 +245,11 @@ int ACA_OVS_L3_Programmer::create_or_update_router(RouterConfiguration &current_
 
           ACA_OVS_L2_Programmer::get_instance().execute_openflow_command(
                   cmd_string, dataplane_programming_time, overall_rc);
-
+          ACA_LOG_DEBUG("Tried to add this flow for ICMP responder: [%s], rc: [%ld]\n",
```

Comment:
Please do not include routing changes to gRPC perf improvement PR.

Reply:
@tianyuan129 , please delete line 248 to 252.

Reply:
Fixed